src/inspector_agent.cc

```diff
@@ -1,5 +1,9 @@
 #include "inspector_agent.h"
 
+#if !HAVE_INSPECTOR
+#error("This file can only be used when the inspector is enabled")
```

Comment:
Since I changed the code to process both "debugger" and inspector properties, now it lives in a dedicated header - I restored the check in the inspector_agent.h